packages/venia-ui/lib/components/CartPage/PriceAdjustments/CouponCode/__tests__/__snapshots__/couponCode.spec.js.snap

```diff
@@ -18,6 +18,7 @@ exports[`disables remove button on click 1`] = `
 exports[`disables submit button on coupon entry 1`] = `
 <form
   className="entryForm"
+  onKeyDown={[Function]}
```

Comment:
These are additions by `informed`.

Reply:
I bet this is related to the new `preventEnter` prop added in `2.4.0`. The previous default behavior allowed enter which is consistent with the default now, so this seems safe.